examples/reshare/src/dkg/actor.rs

```diff
@@ -366,7 +366,15 @@ where
                         }
 
                         if let Some(log) = block.log {
```

Comment:
I suppose we could offer some sort of standard struct for this at some point (but not in this PR).